### ISSUE TITLE
Saving a model for use later

### ISSUE THREAD

**Issue by dberma15** [amirsaffari/online-random-forests#4]
Hi,

Is there a way to save a model and then test it on another set of data for a later use? 
So the steps I'd like to take are:
1) Train the model on the training data
2) Test the model on the test set
3) save the current position of the Random Forest
4) Load that Random Forest
5) Test it again on a different test set

Or is there a way to test it on two test sets stored in two different files and get the metrics for each separately?

**Comment by amirsaffari** [comment]
Hi,

I didn't implement saving and loading for it. However, you could either
- append the two sets together, test them, and split the predictions subsequently to measure performance on both datasets. Or
- change the config file to be able to read a list of files and change the codes [here](https://github.com/amirsaffari/online-random-forests/blob/master/src/Online-Forest.cpp#L113) and [here](https://github.com/amirsaffari/online-random-forests/blob/master/src/Online-Forest.cpp#L151) to enable loading and testing on multiple datasets.
